client/modules/IDE/components/AssetList.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { Link } from 'react-router';
+import InlineSVG from 'react-inlinesvg';
+import prettyBytes from 'pretty-bytes';
+
+import * as AssetActions from '../actions/assets';
+
+const trashCan = require('../../../images/trash-can.svg');
+
+class AssetList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props.getAssets(this.props.username);
+  }
+
+  render() {
+    return (
+      <div className="asset-table-container">
+        <table className="asset-table">
+          <thead>
+            <tr>
+              <th className="asset-list__delete-column" scope="col"></th>
+              <th>Name</th>
+              <th>Size</th>
+              <th>View</th>
+              <th>Sketch</th>
+            </tr>
+          </thead>
+          <tbody>
+            {this.props.assets.map(asset =>
```

Comment:
I agree that it's slow and it may be nice to have a loading state for this view, which could also be used for the `SketchList` view as well.

Also agree that this should handle the case in which there are no assets 😸